SITnett/graphics/style.css

```
@@ -315,13 +315,30 @@ p {
 	position: relative;
 	cursor: pointer;
 	width: 100%;
-	background-color: var(--sit-gunmetal-light);
+	background-color: var(--main-bg-color);
+	border: 2px solid var(--sit-grullo-light);
 }
   
 .card:hover {
 box-shadow: 0 8px 16px 0 rgba(0,0,0,0.2);
 }
 
+/*Litt hårete med store bokstaver i variabelnavn. Stammer fra backend*/
+.card-Kostyme{
```

Dette kan fikses gjennom en innebygget template tag i Django, se under.
```suggestion
.card-kostyme{
```